packages/ndla-ui/src/AudioPlayer/SpeechControl.tsx

```diff
@@ -15,9 +15,10 @@ import Tooltip from "@ndla/tooltip";
 type Props = {
   src: string;
   title: string;
+  label?: string;
 };
 
-const SpeechControl = ({ src, title }: Props) => {
+const SpeechControl = ({ src, title, label = "audio.play" }: Props) => {
```

Comment:
Det er ingen andre plasser vi sender inn nøkkel slik. Enten tenker eg du kan sende inn ferdig rendra label, eller en type med enten audio eller gloss som gyldige verdier. Då blir det "t(`${type}.play`)" i label og aria-label.

Reply:
Hehe, prøvde meg på noe her men mistenkte at det blir for rart og kommer til å bli skutt ned 😅